streamlit_gsheets/gsheets_connection.py

```diff
@@ -36,7 +36,7 @@
 from streamlit.runtime.caching import cache_data
 from streamlit.type_util import convert_anything_to_df, is_dataframe_compatible
 from validators.url import url as validate_url
-from validators.utils import ValidationFailure
+from validators.utils import ValidationError as ValidationFailure
```

Comment:
As a trial, I removed the alias and renamed the handful of instances within the code [in a fork](https://github.com/sfc-gh-dmatthews/gsheets-connection/tree/fix-ruff-linting). I was able to successfully install and run in a clean environment.

`pip install streamlit git+https://github.com/sfc-gh-dmatthews/gsheets-connection/tree/fix-ruff-linting`

Reply:
Wow, so I literally read this as "ValidationError as ValidationError" 😂 

But, that being said, I would prefer Debbie's approach, as it seems more intuitive to just rename the uses, rather than aliasing it as a very similar name

Reply:
👍 I've rewritten failure login into `ValidationError` instead of using `as`